### PR TITLE
[WEB-2382] chore: notifications file rearranging

### PR DESCRIPTION
**Pull request by mathalav55**
### Summary
Rearranged files for notifications

### Reference
[WEB-2382](https://app.plane.so/plane/projects/02c3e1d5-d7e2-401d-a773-45ecba45d745/issues/b5b397c8-31af-4f5b-ba27-28bd562bb96b)

## Release Notes

- **New Features**
  - Introduced a new `NotificationsSidebar` component for improved notification management.
  - Added `NotificationsSidebarRoot` as a wrapper for the `NotificationsSidebar`.

- **Enhancements**
  - Re-exported components for easier access across modules, including `NotificationItem`, `NotificationCardListRoot`, and various notification options.
  - Updated export statements to consolidate access to notification-related functionalities.

- **Bug Fixes**
  - Updated import paths to ensure correct module resolution without altering existing functionality.

These updates enhance the organization and accessibility of notification components, streamlining user interaction with notifications.

### CODE REVIEW

File: web/ce/components/workspace-notifications/sidebar/notification-card/options/read.tsx [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { FC } from "react";
...
 import { MessageSquare } from "lucide-react";
 import { TOAST_TYPE, setToast } from "@plane/ui";
-// components
-import { NotificationItemOptionButton } from "@/components/workspace-notifications";
 // constants
 import { NOTIFICATIONS_READ } from "@/constants/event-tracker";
 // hooks
 import { useEventTracker, useWorkspaceNotifications } from "@/hooks/store";
+// components
+import { NotificationItemOptionButton } from "@/plane-web/components/workspace-notifications";
```
[the remark this answers is not in the document]
Comment (mathalav55) [reply web/ce/components/workspace-notifications/sidebar/notification-card/options/read.tsx:12]: Other imported components are still in core not moved.

File: web/ce/components/workspace-notifications/sidebar/notification-card/options/archive.tsx [hunk 2]
```diff
@@ -4,14 +4,14 @@ import { FC } from "react";
...
-import { NotificationItemOptionButton } from "@/components/workspace-notifications";
 // constants
 import { NOTIFICATION_ARCHIVED } from "@/constants/event-tracker";
 // hooks
 import { useEventTracker, useWorkspaceNotifications } from "@/hooks/store";
+// components
+import { NotificationItemOptionButton } from "@/plane-web/components/workspace-notifications";
 // store
-import { INotification } from "@/store/notifications/notification";
+import { INotification } from "@/plane-web/store/notifications/notification";
```
[the remark this answers is not in the document]
Comment (mathalav55) [reply web/ce/components/workspace-notifications/sidebar/notification-card/options/archive.tsx:14]: Same here. Imported components are still in core.

File: web/core/components/workspace-notifications/sidebar/index.ts [hunk 3]
```diff
@@ -7,4 +7,4 @@ export * from "./header";
 
 export * from "./filters";
 
-export * from "./notification-card";
+// export * from "./notification-card";
```
Comment (rahulramesha) [inline web/core/components/workspace-notifications/sidebar/index.ts:10]: @mathalav55 please check

File: web/core/components/workspace-notifications/sidebar/root.tsx [hunk 4]
```diff
@@ -1,105 +1,3 @@
...
-              </div>
-            )}
-          </>
-        )}
-      </div>
-    </div>
-  );
-});
+import { NotificationsSidebar } from "@/plane-web/components/workspace-notifications";
+export const NotificationsSidebarRoot = () => <NotificationsSidebar/>
```
Comment (rahulramesha) [inline web/core/components/workspace-notifications/sidebar/root.tsx:3]: This Change feel unnecessary. Please check

**Pull request closed without merging**